### PR TITLE
[AllBundles] added sf4 composer

### PR DESCRIPTION
**Pull request by deZinc**
WIP to use SF4 in composer

### CODE REVIEW

**Comment by jockri** [comment]
@deZinc can you remove all config changes and only keep the changes in composer.json?

**Comment by deZinc** [comment]
@jockri : done

File: composer.json [hunk 1]
```diff
@@ -64,14 +64,17 @@
         "knplabs/knp-gaufrette-bundle": "~0.1",
         "symfony-cmf/routing-bundle": "~2.0",
         "fpn/doctrine-extensions-taggable": "~0.9",
+        "twig/twig": "^1.12|^2.0",
```
Comment (acrobat) [inline composer.json]: This line should be removed, correct `twig/twig` dependency is set on line 68/69

File: composer.json [hunk 2]
```diff
@@ -64,14 +64,17 @@
...
         "twig/twig": "^1.36|^2.6",
         "twig/extensions": "~1.0",
         "egulias/email-validator": "^1.2.8|^2.0",
         "box/spout": "^2.5",
         "ruflin/elastica": "^5.1|^6.0",
         "behat/transliterator": "~1.2",
-        "defuse/php-encryption": "v2.1.0"
+        "defuse/php-encryption": "v2.1.0",
+        "symfony/maker-bundle": "^1.9",
+        "symfony/monolog-bundle": "^3.3"
```
Comment (acrobat) [inline composer.json]: Do we need the monolog bundle? As it was removed in #2061

File: composer.json [hunk 3]
```diff
@@ -64,14 +64,17 @@
...
         "sensio/generator-bundle": "~3.0",
         "twig/twig": "^1.36|^2.6",
         "twig/extensions": "~1.0",
         "egulias/email-validator": "^1.2.8|^2.0",
         "box/spout": "^2.5",
         "ruflin/elastica": "^5.1|^6.0",
         "behat/transliterator": "~1.2",
-        "defuse/php-encryption": "v2.1.0"
+        "defuse/php-encryption": "v2.1.0",
+        "symfony/maker-bundle": "^1.9",
```
Comment (acrobat) [inline composer.json]: Do we need the maker bundle now? I don't think we use a maker in the current code

File: composer.json [hunk 4]
```diff
@@ -129,4 +132,4 @@
             "dev-master": "5.2-dev"
         }
     }
-}
+}
```
Comment (acrobat) [inline composer.json]: Missing newline

**Pull request merged**